app/models/feed.rb

```diff
@@ -24,9 +24,14 @@ def from_redis(limit, max_id, since_id)
   end
 
   def from_database(limit, max_id, since_id)
-    Status.as_home_timeline(@account)
-          .paginate_by_max_id(limit, max_id, since_id)
-          .reject { |status| FeedManager.instance.filter?(:home, status, @account.id) }
+    for attempt in 0..20 do
```

Comment:
`for attempt in 0..20 do` -> `21.times do`

Reply:
Right, thanks! This commit is not needed, though, I just tried to avoid the test failures, but I still don't understand it. I will drop that commit.